indico/modules/events/papers/templates/display/call_for_papers.html

```diff
@@ -0,0 +1,75 @@
+{% extends 'layout/conference_page_base.html' %}
```

Comment:
Why not inherit from 'events/papers/_base.html'? that already sets the title block